### PR TITLE
Add workflow that publishes a GeoTIFF

### PR DESCRIPTION
**Pull request by JakobMiksch**
Adds a basic round-trip of a workflow that downloads a GeoTIFF and publishes it to GeoServer.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,3 +14,9 @@ docker-compose up
 # download latest images if they have changed
 docker-compose pull
 ```
+
+## Workflows
+
+The directory `workflows` contains workflows that can be send to the `dispatcher`:
```
Comment (weskamm) [inline README.md:20]:
```suggestion
The directory `workflows` contains workflows that can be sent to the `dispatcher`:
```

File: docker-compose.yml [hunk 2]
```diff
@@ -25,6 +29,37 @@ services:
       - RESULTSQUEUE=${RABBITMQ_RESULTSQUEUE}
       - WORKERQUEUE=dispatcher
 
+  worker-geoserver-publish-geotiff:
+    image: ghcr.io/klips-project/mqm-worker/geoserver-publish-geotiff:latest
+    restart: unless-stopped
+    volumes:
+      - tmp-data:/home/data:Z
+    depends_on:
+        - rabbitmq
```
Comment (weskamm) [inline docker-compose.yml:38]:
```suggestion
      - rabbitmq
```

File: docker-compose.yml [hunk 3]
```diff
@@ -25,6 +29,37 @@ services:
       - RESULTSQUEUE=${RABBITMQ_RESULTSQUEUE}
       - WORKERQUEUE=dispatcher
 
+  worker-geoserver-publish-geotiff:
```
Comment (weskamm) [inline docker-compose.yml:32]: not that important, but until now we always used the same name for the worker image, the docker container and the `WORKERQUEUE`. I would like to keep it in sync that way, so please rename the container

File: docker-compose.yml [hunk 4]
```diff
@@ -25,6 +29,37 @@ services:
...
+    - RABBITHOST=${RABBITMQ_HOSTNAME}
+    - RABBITUSER=${RABBITMQ_DEFAULT_USER}
+    - RABBITPASS=${RABBITMQ_DEFAULT_PASS}
+    - RESULTSQUEUE=${RABBITMQ_RESULTSQUEUE}
+    - WORKERQUEUE=geoserver-publish-geotiff
+    - GEOSERVER_REST_URL=http://${GEOSERVER_HOSTNAME}:8080/geoserver/rest/
+    - GEOSERVER_USER=${GEOSERVER_USER}
+    - GEOSERVER_PASSWORD=${GEOSERVER_PASSWORD}
+
+  worker-download:
```
Comment (weskamm) [inline docker-compose.yml:49]: see comment above, please use identical naming for container, image and workerqueue `download-new-data-from-url`.
Of course if you would like to refactor that workers name to something easier, i wont hold you back :-)

File: docker-compose.yml (hunk 3 above)
Comment (JakobMiksch) [reply docker-compose.yml]: I agree, I changed it

File: docker-compose.yml (hunk 4 above)
Comment (JakobMiksch) [reply docker-compose.yml]: Indeed I did not like the "new" in the name `download-new-data-from-url`, so I renamed it. But I agree that consistency is more important here!

Comment (weskamm) [reply docker-compose.yml]: i have renamed that worker in another PR, so this will need adjustment soon

**Review by weskamm — APPROVED** [review]

**Pull request merged**